spec/requests/api/cycles_spec.rb

```diff
@@ -42,4 +42,42 @@
       end
     end
   end
+
+  path '/api/cycles/{id}' do
+    get 'Returns Cycle details' do
+      tags 'Cycles'
+      description 'Retrieves the details of a cycle'
+      parameter name: :curriculum_id, in: :path, type: :string
+      parameter name: :id, in: :path, type: :string
+      produces 'application/json'
+      operationId 'getCycle'
+
+      let(:curriculum_id) { create(:curriculum).id }
+      let(:cycle) { create(:cycle, curriculum_id:) }
+      let(:id) { cycle.id }
+
+      response '200', 'Success' do
+        schema type: :object,
+               properties: {
+                 id: { type: :integer },
+                 name: { type: :string },
+                 order_number: { type: :integer },
+                 learning_goals_description: { type: :string },
+                 challenge_description: { type: :string },
+                 boilerplate_url: { type: :string }
+               }
+
+        run_test!
+      end
+
+      response '401', 'Unauthorized', skip_before: true do
```

Comment:
a cycle information should be public to users that are not authenticated, following [this PR's guidelines](https://github.com/fintual-oss/paraffin-back/pull/10), therefore you should delete this test

Reply:
done! thanks